### PR TITLE
Adds a way to have a prefered security department.

### PR DESCRIPTION
**Pull request by lzimann**
:cl: Lzimann
add: Now you can choose what department you want to be as security! (This may not be completly reliable).
/:cl:

As suggested [here](https://tgstation13.org/phpBB/viewtopic.php?f=9&t=9348). This should work, won't always put you there but should help.

### CODE REVIEW

**Comment by Armhulen** [comment]
:tada: !!!

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -64,6 +64,8 @@ var/list/preferences_datums = list()
 	var/list/features = list("mcolor" = "FFF", "tail_lizard" = "Smooth", "tail_human" = "None", "snout" = "Round", "horns" = "None", "ears" = "None", "wings" = "None", "frills" = "None", "spines" = "None", "body_markings" = "None", "legs" = "Normal Legs")
 
 	var/list/custom_names = list("clown", "mime", "ai", "cyborg", "religion", "deity")
+	var/prefered_security_department = "Random"
```
Comment (RemieRichards) [inline code/modules/client/preferences.dm]: defines.

Comment (lzimann) [inline code/modules/client/preferences.dm]: yea i was just too lazy to do them

**Comment by Shapsy** [comment]
The only reason to use this is so you can raid cargo and research for items without consequence more consistently, all this will do is leave engineering consistently unguarded since it doesn't have fun stuff to steal and ensure that  shitters who want free stuff from cargo and science get first dibs on those positions

Leave it random

**Comment by SweaterKittens** [comment]
I highly disagree. This is a huge quality of life improvement for security officers and will encourage people to play it, knowing they can be a part of their favorite department. Yes, things like science will be staffed more frequently, but I think you're jumping to the worst conclusion.

Back when I mained sec I would've LOVED something like this so that I could be a part of medical each round. If people are being shitty with their access and "commandeering" shit that doesn't belong to them just because they're wearing red, that's a player problem. Department sec guard are supposed to have access so they can protect the department better, not because they're actually a part of that department.  I'll happily deal out secbans to people who role Science Sec and use it to push the scientists out and take RnD over.

**Comment by BeeSting12** [comment]
moonlighting is NOT okay

File: code/modules/client/preferences_savefile.dm [hunk 2]
```diff
@@ -306,6 +306,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	S["cyborg_name"]		>> custom_names["cyborg"]
 	S["religion_name"]		>> custom_names["religion"]
 	S["deity_name"]			>> custom_names["deity"]
+	S["prefered_security_department"] >> prefered_security_department
```
Comment (Core0verload) [inline code/modules/client/preferences_savefile.dm]: It wouldn't cause any problems with old saves, right?

Comment (Incoming5643) [inline code/modules/client/preferences_savefile.dm]: It shouldn't, you only need to worry about that when changing a save option that already existed.

**Comment by Saegrimr** [comment]
> all this will do is leave engineering consistently unguarded since it doesn't have fun stuff to steal 

You mean the easy insulateds and tools?

Come on man, Engineering would be the easiest pick for looters because you grab you shit and everybody leaves. People who get randomly selected as engisec already fucking throw away their armband because once the engine is set up NOBODY sticks around in engineering. Cargo's job is in Cargo and they do it all round, Science's job is in Science and they do that all round, Engineering spends 5 minutes setting up the engine and then they all scatter to the four winds looking for something else to do.

Nobody wants to guard an empty department.

File: code/modules/jobs/job_types/security.dm [hunk 3]
```diff
@@ -181,40 +181,44 @@ Security Officer
 	L |= ..() | check_config_for_sec_maint()
 	return L
 
-var/list/sec_departments = list("engineering", "supply", "medical", "science")
+var/list/available_depts = list("Engineering", "Supply", "Medical", "Science")
```
Comment (RemieRichards) [inline code/modules/jobs/job_types/security.dm]: use the defines you made.

Comment (lzimann) [inline code/modules/jobs/job_types/security.dm]: That's odd, I swear I've edited these.

**Comment by bgare89** [comment]
Good Job LeoZ on the quick turn around on this. Pref Sec is going to be huge.

**Pull request merged**